dev/source/docs/building-the-code.rst

```diff
@@ -15,18 +15,36 @@ Setting up the Build Environment
 - :ref:`Setup the Build Environment on Windows <building-setup-windows>`
 - :ref:`Setup the Build Environment on MacOSX <building-setup-mac>`
 
+
 Building / Compiling
 --------------------
 
-**Linux / MacOSX users:**
+Once the build environment is set up as above, run the following commmands prior to the initial build. (For Windows users, this is under WSL, as set up above):
 
-- Linux and MacOSX users should build with waf as described in `BUILD.md <https://github.com/ArduPilot/ardupilot/blob/master/BUILD.md>`__.
+- `git submodule init`
```

Comment:
remove

Reply:
It seems this is required

Reply:
This is required to avoid the following error message:

```c
Build failed
 -> task in 'dronecan' failed (exit status 2):
        {task 140083940872752: dronecangen  -> }
 (run with -v to display more information)
 ```